static/css/taxbrain.css

```diff
@@ -8440,6 +8440,18 @@ label {
   border-right: 2px solid #dddddd;
   padding-left: 30px;
 }
+.table-check-all {
```

Comment:
I'm not much of a css person, but I wanted to point out that my understanding is that the CSS is generated from the `less` files. Would it make sense to add to the appropriate `less` file and then have the new version of `taxbrain.css` generated from that?